tests/integration/build_invoke/dotnet/test_build_invoke_dotnetcore3_1.py

```diff
@@ -14,43 +14,51 @@
 class BuildInvoke_dotnetcore3_1_cookiecutter_aws_sam_hello_dotnet(BuildInvokeBase.DotNetCoreExtraRerunBuildInvokeBase):
     use_container = False
     directory = "dotnetcore3.1/hello"
+    should_test_lint = False
```

Comment:
I understand why the python3.10 are marked as `should_test_lint = False`, but it isn't clear why the rest are marked as shouldn't be linted. Could you describe the reason for them? You don't have to list a reason for each but understand the buckets of reasons would be helpful.

Reply:
Added a note at the top of files (nodejs12.x and dotnet3.1)